src/Apps/PrivateSaleConditions/PrivateSaleConditions.tsx

```diff
@@ -0,0 +1,12 @@
+import { Flex, Spacer, Text } from "@artsy/palette"
+
+export const PrivateSaleConditions = () => {
```

Comment:
Our current legal pages on artsy.net are implemented as a [page](https://github.com/artsy/gravity/blob/4e763c73609097b84dfe9c970e37ad96c1ddcd5a/app/models/domain/page.rb) in Gravity (content can be entered from [the old admin](https://admin.artsy.net/pages) UI), and is fetched and displayed in the [Page app](https://github.com/artsy/force/blob/5ddf990c01626222c1dce1dcbc4408ce530912cc/src/Apps/Page/PageApp.tsx).

This condition of sale for private sales requires authentication that's different than the other public legal agreements, but I wonder if it can still be implemented in the Page app.